Functions.Templates/Templates/TimerTrigger-CSharpExcel/function.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "bindings": [
```

Comment:
Function.json does not match the run.csx for TimerTrigger-CSharpExcel. Code has arguments from bindings that are not present in function.json